charts/gateway-helm/crds/generated/gateway.envoyproxy.io_backendtrafficpolicies.yaml

```diff
@@ -947,6 +947,11 @@ spec:
                               - requests
                               - unit
                               type: object
+                            name:
```

Comment:
guessing there should be more changes in the CRD, tied to the `mergeKey` ?

Reply:
`patchMergeKey` won't affect CRD files.